### ISSUE TITLE
Add graphite to HD pi-k8s

### ISSUE THREAD

**Issue by gaf3** [gaf3/pi-k8s-automation#27]
Basic install, preferably on k8s with local mounts.

**Comment by gaf3** [comment]
https://graphite.readthedocs.io/en/latest/install.html

**Comment by gaf3** [comment]
Error when installing on pi:
```
HEAD is now at 8d5363c Fix for failing test on node 0.10
npm WARN deprecated boom@2.10.1: This version is no longer maintained. Please upgrade to the latest version.
npm WARN deprecated cryptiles@2.0.5: This version is no longer maintained. Please upgrade to the latest version.
npm WARN deprecated hoek@2.16.3: This version is no longer maintained. Please upgrade to the latest version.
npm ERR! cb() never called!

npm ERR! This is an error with npm itself. Please report this error at:
npm ERR!     <https://npm.community>

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2018-12-20T18_45_25_315Z-debug.log
```

**Comment by gaf3** [comment]
https://github.com/pi-k8s-fitches/graphite
